index.js

```diff
@@ -1,15 +1,52 @@
 module.exports = ({
-  user = (ctx) => `You're the following permissions: ${ctx.command.userPerms.join(', ')}`,
-  my = (ctx) => `I'm missing the following permissions: ${ctx.command.myPerms.join(', ')}`
+  humanReadable = {
```

Comment:
Why is this an object passed to the method? Just make it a constant

Reply:
language support??!

Reply:
alright

Reply:
You should still make the default a constant, and possibly export it too, and just make the humanReadable = the constant.

Reply:
For better language support, wouldn't you want to be able to set `humanReadable` to a function as well? This would mean that you could have different languages for different guilds without rewriting the middleware to better suit the bot.